features/product/viewing_products/viewing_product_via_slug.feature

```diff
@@ -0,0 +1,15 @@
+@viewing_products
+Feature: Viewing a product details using slug
```

Comment:
Let's put it into https://github.com/Sylius/Sylius/blob/master/features/product/viewing_products/accessing_product_page_via_permalink.feature file, just below the already existing scenario. Also, can you add `no-api` tag to the scenario that is already there?